docs/3.main-concepts/7.humanhandoff.mdx

```diff
@@ -8,16 +8,27 @@ menu: 3. Main Concepts
 
 As bots aren’t perfects, we allow to transfer a conversation into an agent.
 
-To configure transfer the conversation into an agent, you must provide the `session` parameters and then the `queue_name` and `next_action`, which are optional, but recommended.
+To configure transfer the conversation into an agent, you must provide the `session` parameters and then the `queueName` and `onFinish`, which are optional, but recommended.
 
-The `queue_name` is the name of the queue where you want to transfer the conversation. If a queue with that name doesn't exist, it will transfer the chat into a random open queue.
+The `queueName` is the name of the queue where you want to transfer the conversation. If a queue with that name doesn't exist, it will transfer the chat into a random open queue.
 
-The `next_action` allows you to specify which is going to be the next action that the bot will follow after the conversation with the agent has finished.
+The `onFinish` allows you to specify a payload or a path in order to trigger the next action that the bot will follow after the conversation with the agent has finished.
 
-Here’s a simple example of how to transfer the conversation into the queue 'Support_Queue' and the next step of the bot will be an action called 'say_goodbye'.
+Here’s a simple example of how to transfer the conversation into the queue `Support_Queue` and the next step of the bot will be the action called `SayGoodbye` which will be triggered after receiving the payload `sayGoodbye` (which has to be defined in your `routes.js` file).
```

Comment:
"`Support_Queue` and the" => . "`Support_Queue`.  The".

lo bue si bre, 2 veces bue: http://www.ghostlittle.com/blog/examples-why-short-sentences-are-better-than-long-sentences

Reply:
fixed! nice post!

Reply:
not yet changed

Reply:
just done the commit!